src/app/types.json

```diff
@@ -173,5 +173,11 @@
       "audience": "end user",
       "object": "end user’s live service usage"
     }
-  }
+  },
+  "Ad Publishing Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "acceptable and unacceptable usage"
+    }
```

Comment:
The json is not valid, there is a } missing at end.

Reply:
Added that extra } !